README.md

```diff
@@ -1,3 +1,5 @@
+[![CircleCI](https://circleci.com/gh/mozilla/addons.svg?style=svg)](https://circleci.com/gh/mozilla/addons)
```

Comment:
I like to put badges below the title but that's a matter of preferences I think.